setup.py

```diff
@@ -33,22 +33,7 @@
 
 # For Windows install PyWin32 if not already installed.
 if sys.platform.startswith('win'):
-    # Windows support depends on pefile library.
-    # ::TODO:: #1920 require a specific version minimum
-    # REQUIREMENTS.append('pefile')
-    # ::TODO:: #1920 future is currently only needed for the included pefile,
-    # remove 'future' as requirement when pypi pefile is used
-    REQUIREMENTS.append('future')
```

Comment:
I'm afraid, you lost pefile and future?!

Reply:
We don't directly depend on future, and pefile is already included above.